Shared/Enums/Status.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Shared.Enums;
+
+public enum Status: byte
+{
+    Active=0,
```

Comment:
Please add a space before and after assignment for better readability
Active = 0 ,

Reply:
Fixed